### PR TITLE
feat/FAT-660 Edit device name on LLD

### PR DESCRIPTION
**Pull request by juan-cortes**
### 📝 Description

Let users rename their device from My Ledger with a more or less accurate UI (this no-man's land between UI versions is a pain) but functional nonetheless. It also adds the auto fetching of the name upon accessing the manager which should avoid the user prompt. Pay special attention to this because if it somehow triggers the prompt in some cases we need to roll this back.

### ❓ Context

- **Impacted projects**: `ledger-live-desktop` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/FAT-660` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/4631227/219071251-62c80e2d-f310-462a-b689-552ee1ef06ad.mov

image
image

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/CustomImageManagerButton.tsx [hunk 1]
```diff
@@ -12,12 +13,11 @@ const CustomImageManagerButton: React.FC<Record<string, never>> = () => {
   }, []);
   return (
     <Flex flexDirection="row" columnGap={3} alignItems="center">
-      <Icons.ToolsMedium color="neutral.c80" size={16} />
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/CustomImageManagerButton.tsx:15]: are you sure about this icon? I feel like it was used as a placeholder in Figma...
anyway we will do a pixel polish soon so this doesn't matter

Comment (juan-cortes) [reply apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/CustomImageManagerButton.tsx:15]: I'm not sure about anything, I just followed the figma

**Review by valpinkman — APPROVED** [review]
LGTM

**Pull request merged**